src/registry/interfaces/vault-registry-metadata.interface.ts

```diff
@@ -1,4 +1,4 @@
-export interface VaultRegistryV2Metadata {
+export interface VaultRegistryMetadata {
   name?: string;
```

Comment:
name, protocol, behavior are now all considered required on v2
client and other are optionals